.gitignore

```diff
@@ -5,7 +5,7 @@
 /vendor/
 ###< symfony/framework-bundle ###
 
-/.idea/
```

Comment:
Keep the idea please. I've it the global git config but most people don't

Reply:
sure, no problem.